app/javascript/components/public_health/PatientsTable.js

```diff
@@ -60,7 +61,7 @@ class PatientsTable extends React.Component {
           { field: 'reason_for_closure', label: 'Reason for Closure', isSortable: true, tooltip: null },
           { field: 'closed_at', label: 'Closed At', isSortable: true, tooltip: null, filter: this.formatTimestamp },
           { field: 'transferred_at', label: 'Transferred At', isSortable: true, tooltip: null, filter: this.formatTimestamp },
-          { field: 'latest_report', label: 'Latest Report', isSortable: true, tooltip: null, filter: this.formatTimestamp },
```

Comment:
My PR (SARAALERT-654) touches this code, so which ever one of our PRs gets merged in first, the other one will probably require some very minor tweaking.